metacat-common/src/main/java/com/netflix/metacat/common/dto/AuditDto.java

```diff
@@ -26,6 +29,9 @@
 @SuppressWarnings("unused")
 @Data
 @EqualsAndHashCode(callSuper = false)
```

Comment:
Why don't we want to call super here? I would think it would be important to know that the base class members are equal as well, no?